test/EndToEnd/tests/InstallPackageTest.ps1

```diff
@@ -2592,38 +2488,6 @@ function Test-NonFrameworkAssemblyReferenceShouldHaveABindingRedirect
     Assert-BindingRedirect $p app.config System.Web.Razor '0.0.0.0-3.0.0.0' '3.0.0.0'
 }
 
-# Temporarily comment out the test that's hang VS (during Javascript project creation)
-# NuGet is not involved in that step. We may need to update the template.
-function InstallPackageIntoJavaScriptApplication {
-    # Windows 8.x/Phone tests irrelevant post-VS14
-    [SkipTestForVS15()]
-    param()
-
-    # Arrange
-    $p = New-JavaScriptApplication
-
-    # Act
-    Install-Package jQuery -ProjectName $p.Name
-
-    # Assert
-    Assert-Package $p "jQuery"
-}
-
-function Test-InstallPackageIntoJavaScriptWindowsPhoneApp {
-    # Windows 8.x/Phone tests irrelevant post-VS14
-    [SkipTestForVS15()]
-    param()
-
-    # Arrange
-    $p = New-JavaScriptWindowsPhoneApp81
-
-    # Act
-    Install-Package jQuery -ProjectName $p.Name
-
-    # Assert
-    Assert-Package $p "jQuery"
-}
-
 function Test-InstallPackageIntoNativeWinStoreApplication {
     [SkipTestForVS15()]
```

Comment:
This one probably needs removed too. 

Find the references of SkipTestForVS15 and remove them.